File: Programmers Level1/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(long long n) {
	vector<int> answer;
	// 12345
	string cnt = to_string(n);
	reverse(cnt.begin(), cnt.end());
	for (int i = 0; i < cnt.size(); i++) {
		answer.push_back(cnt[i] - '0');
	}
	return answer;
}

File: Problem/src/자연수 뒤집어 배열로 만들기.kt
class Solution {
    fun solution(n: Long): ArrayList<Int> {
        var answer = ArrayList<Int>()
        var string = n.toString()
        string = string.reversed()
        string.forEach {
            answer.add(it-'0')
        }
        var a = ArrayList<Int>()
        a.sortedDescending()
        print(answer)
        return answer
    }
}